### PR TITLE
Extensive changes and troubleshooting for compatibility with the new version of the template

### PR DESCRIPTION
**Pull request by M0H4MM4D0UTIS**
1- add new method for __NEXT_DATA__ 
2- Fix a problem that shows only 3 genres (and add test for this)
3- more fix with new tamplate

Explain only 3 genres problem:
sample:
https://www.imdb.com/title/tt1375666/

old result:
array (size=3)
  0 => string 'Action'
  1 => string 'Adventure'
  2 => string 'Sci-Fi'

new result:
array (size=4)
  0 => string 'Action'
  1 => string 'Adventure'
  2 => string 'Sci-Fi'
  3 => string 'Thriller'

### CODE REVIEW

**Comment by M0H4MM4D0UTIS** [comment]
jsonLD only show 3 result

File: tests/TitleTest.php [hunk 1]
```diff
@@ -496,10 +497,11 @@ public function testLanguages_detailed()
     // Primary match is to the genre listing just under the title, which this tests
     public function testGenres_multiple()
     {
-        $imdb = $this->getImdb();
+        $imdb = $this->getImdb('1375666');
```
Comment (tboothman) [inline tests/TitleTest.php:500]: Could you change this to nausicaa / 0087544. It's got 4 genres and the tests already have this page downloaded

Comment (M0H4MM4D0UTIS) [reply tests/TitleTest.php:500]: Yes. Thanks for your suggestion. Done

**Comment by M0H4MM4D0UTIS** [comment]
@tboothman
I fixed all the problems and now it is ready to integrate :)

**Pull request merged**